### ISSUE TITLE
Track automated and manual assessment score over time

### ISSUE THREAD

**Issue by nkkl** [18F/a11y-metrics#6]
E.g. if a site has 8 a11y failures when we first survey it and 2 a11y failures when we come back a month later, that should be reflected as positive progress. Even though they're not at 0, we can look at the trend and feel good assuming that they will get to 0 soon.

Conversely, if a site moves from 2 failures to 10, we know that something bad has happened and the team may need some support.

**Comment by toolness** [comment]
How frequently do you think that the assessment should be tracked?  Once per day?  week?  month?

One easy-ish way to do this would just be to make sure that we re-run `stats.js` at whatever this interval is, and commit `stats.csv` to the repository.  Then we could track the scores over time by writing a script that loads `stats.csv` at previous points in history and generates time series data from it.

Or alternatively we could just make `stats.js` append the current date to the CSV filename, which would obviate the need for rifling through GitHub history.

Alternatively, if we don't trust ourselves to run the script manually (or if it's too much of a hassle), we could potentially set up a [Travis cron job](https://docs.travis-ci.com/user/cron-jobs/) or some sort of worker dyno on cloud.gov that re-runs the script at regular intervals, either committing results to a database or to the GitHub repository.

**Comment by nkkl** [comment]
Once a month is totally fine. Not more than once a sprint.

And I'd only capture `n` and `n-1`, so we just get a "did you improve/decline since last time". Nothing convoluted.

**Comment by nkkl** [comment]
We could do a counter of sorts — a useful thing to know is how long a project has sat without taking any action on the open a11y issues. For example, a project that pushed a bunch of new code in the last sprint that happened to cause some aXe failures isn't nearly as worrying to me as a project that has had a11y bugs sitting on them for months.
